.github/workflows/clojars.yml

```diff
@@ -0,0 +1,35 @@
+name: Clojure CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
```

Comment:
Deploying to Clojars on every push to master (hey, shouldn't that be `main`?) is a specific choice, but perhaps not the correct one here.

Reply:
This is whenever a tag is pushed. I think it makes sense here